server/controllers/interview.js

```diff
@@ -0,0 +1,109 @@
+// connect to the Interview Model
+let Interview = require('../models/interview');
+let User = require('../models/user');
```

Comment:
These can be const

Reply:
The variables that don't require "let" have been changed to "const"